src/components/toolbar-menu/toolbar-list.html

```diff
@@ -1,15 +1,20 @@
 <div class="btn-group" dropdown>
   <button type="button" dropdown-toggle class="btn dropdown-toggle btn-default"
-          ng-class="{disabled: !toolbarList.enabled}" title="{{toolbarList.title}}">
-    <i class="{{toolbarList.icon}}" style="margin-right: 5px;" ng-if="toolbarList.icon"></i>
-    {{toolbarList.title}}
+          ng-class="{disabled: !vm.toolbarList.enabled}" title="{{vm.toolbarList.title}}">
+    <i class="{{vm.toolbarList.icon}}" style="margin-right: 5px;" ng-if="vm.toolbarList.icon"></i>
+    {{vm.toolbarList.title}}
```

Comment:
I'm wondering because of how these components are used in MiQ, since they are not being started up once but possibly many different times in different parts of the application.  This being the case we might want to use `ng-bind` or `ng-cloak` instead of `{{myVar}}` because there isn't just one startup but multiple small ones so possibility of angular not being fully loaded when we view the page is higher than with a normal SPA that only loads the app once.
wdyt?